### PR TITLE
workaround for wallpaper module cannot parse URL well

### PR DESCRIPTION
**Pull request by notmikeb**
when running bingo, its wallpaper failed to recognize the filepath and show 'unable to set wallpaper'
make a workaround that add a end string of '/2020.jpg' to avoid wallpaper failure

C:\Users\xxx\bingo>go run main.go
[bingo] 2020/06/08 23:35:27 [INF] url:'"https://www.bing.com/th?id=OHR.LionSurfing_ROW9727597970_1920x1080.jpg&rf=LaDigue_1920x1080.jpg&pid=hp"'
[bingo] 2020/06/08 23:35:27 [INF] updating wallpaper, url: "https://www.bing.com/th?id=OHR.LionSurfing_ROW9727597970_1920x1080.jpg&rf=LaDigue_1920x1080.jpg&pid=hp", copyright: "Galápagos sea lion off the shore of Fernandina Island, Galápagos Islands, Ecuador (© Tui De Roy/Minden Pictures)"
[bingo] 2020/06/08 23:35:27 [ERR] unable to set wallpaper: open C:\Users\dalon\AppData\Local\Temp\th?id=OHR.LionSurfing_ROW9727597970_1920x1080.jpg&rf=LaDigue_1920x1080.jpg&pid=hp: The filename, directory name, or volume label syntax is incorrect.
[bingo] 2020/06/08 23:35:27 [INF] wallpaper will be updated again daily at 08:00
exit status 2

### CODE REVIEW

**Comment by kkyr** [comment]
Thanks for the PR.

There's an issue since the resulting file name contains an illegal character (`?`). I'll open a PR with the wallpaper library to fix it.

Meanwhile I'll accept this as a temporary workaround.

File: main.go [hunk 1]
```diff
@@ -84,7 +84,9 @@ func setBingWallpaper() {
 		log.Printf("[ERR] unable to retrieve Bing image of the day: %v", err)
 		return
 	}
-
+	// append a dummy appendix for wallpaper module to recognize the filename
+	log.Printf("[INF] url:'%q' ", image.URL)
```
Comment (kkyr) [inline main.go]: Can you remote this line please, it's logged a couple lines below anyway.

**Review by notmikeb — COMMENTED** [review]
i change this ~~

File: main.go (hunk 1 above)
Comment (notmikeb) [reply main.go]: okay. done

**Comment by kkyr** [comment]
Thank you

**Pull request merged**